### ISSUE TITLE
You have validator issues

### ISSUE THREAD

**Issue by delg0030** [isabelle6/portfolio#2]
Please review the following issues noticed by Validator and make those corrections :)
- The web page should have a title that describes topic or purpose (8 results)
- Text elements should have a reasonable contrast ratio (1 result)
